apps/web/src/components/Dashboard/Templates.tsx

```diff
@@ -28,43 +59,128 @@ export function TemplatesContent({
   teamId: string
 }) {
   const [templates, setTemplates] = useState<Template[]>([])
+  const [currentTemplate, setCurrentTemplate] = useState<Template | null>(null)
+  const [isPublishTemplateDialogOpen, setIsPublishTemplateDialogOpen] =
+    useState(false)
+  const [isDeleteTemplateDialogOpen, setIsDeleteTemplateDialogOpen] =
+    useState(false)
 
   useEffect(() => {
-    function f() {
-      const accessToken = user.accessToken
-      if (accessToken) {
-        fetchTemplates(accessToken, teamId).then((newTemplates) => {
-          if (newTemplates) {
-            setTemplates(newTemplates)
-          }
+    async function fetchTemplates() {
```

Comment:
have u considered trying the [useQuery](https://tanstack.com/query/latest/docs/framework/react/overview) hook?

Reply:
I know about it, but it will be in the new web app

Reply:
I see `Template Name` in CLI and in SDK it isn't really exposed